ios/Runner/Info.plist

```diff
@@ -83,5 +83,10 @@
 			<string>https://hypha.earth</string>
 			<string>https://hyphawallet.page.link</string>
 		</array>
+		<key>LSApplicationQueriesSchemes</key>
+        <array>
+          <string>sms</string>
```

Comment:
Nik, is this how this is added to IOS? The package suggested so. 

iOS 
Add any URL schemes passed to canLaunchUrl as LSApplicationQueriesSchemes entries in your Info.plist file, otherwise it will return false.

Example:

<key>LSApplicationQueriesSchemes</key>
<array>
  <string>sms</string>
  <string>tel</string>
</array>